avocado_vt/loader.py

```diff
@@ -79,6 +79,28 @@ def arch_listing(config, guest_name_parser=None):
     LOG.debug("")
 
 
+def get_vt_test_tags(vt_params):
+    """
+    Returns the test categories based on a `tags` vt param
+
+    :rtype: dict
+    """
+    tags = vt_params.get('tags', '').strip()
+    result = {}
+    for tag in tags.split(','):
+        if not tag:
+            continue
+        if ':' in tag:
+            key, val = tag.split(':', 1)
+            if key in result:
+                result[key].add(val)
+            else:
+                result[key] = set([val])
+        else:
+            result[tag] = None
```

Comment:
How about using an empty set here? It will make the types more uniform.

Reply:
Yes, makes sense, will update in the next version, thanks.